import_map.json

```
@@ -1,6 +1,7 @@
 {
   "imports": {
     "mqtt": "npm:mqtt@4.3.7",
-    "mariadb": "npm:mariadb@3.1.1"
+    "mariadb": "npm:mariadb@3.1.1",
+    "oak": "https://deno.land/x/oak/mod.ts"
```

It's an deno/web convention to import filenames and with version number. Therefore the importmap needs to be this.


```suggestion
    "oak/": "https://deno.land/x/oak@v12.4.0/"
```

therefore importing needs to be 

```ts
import { module } from "oak/mod.ts";
```